src/DetailsView/components/test-step-view.tsx

```diff
@@ -1,26 +1,27 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { Spinner, SpinnerSize } from 'office-ui-fabric-react';
-import * as React from 'react';
-
 import { AssessmentDefaultMessageGenerator } from 'assessments/assessment-default-message-generator';
 import { AssessmentsProvider } from 'assessments/types/assessments-provider';
 import { Requirement, VisualHelperToggleConfig } from 'assessments/types/requirement';
-import { ContentPanelButton, ContentPanelButtonDeps } from 'views/content/content-panel-button';
-import { CollapsibleComponent } from '../../common/components/collapsible-component';
-import { GuidanceTags, GuidanceTagsDeps } from '../../common/components/guidance-tags';
+import { CollapsibleComponent } from 'common/components/collapsible-component';
+import { GuidanceTags, GuidanceTagsDeps } from 'common/components/guidance-tags';
 import {
     AssessmentNavState,
     GeneratedAssessmentInstance,
     ManualTestStepResult,
-} from '../../common/types/store-data/assessment-result-data';
-import { FeatureFlagStoreData } from '../../common/types/store-data/feature-flag-store-data';
-import { PathSnippetStoreData } from '../../common/types/store-data/path-snippet-store-data';
-import { DictionaryStringTo } from '../../types/common-types';
+} from 'common/types/store-data/assessment-result-data';
+import { FeatureFlagStoreData } from 'common/types/store-data/feature-flag-store-data';
+import { PathSnippetStoreData } from 'common/types/store-data/path-snippet-store-data';
+import { Spinner, SpinnerSize } from 'office-ui-fabric-react';
```

Comment:
this kind of changes (re-alphabetize imports) always strike me as odd... shouldn't our tslint task catch this?
(btw, I'm not asking that you revert this part... it's just a though)

Reply:
Oh... I just notice you chage the type of path on the imports.... that explains why the office fabric import got push down... my bad.

Reply:
haha yeah, I noticed that as a part of doing the work and figured why not?